src/nvim/api/private/helpers.c

```diff
@@ -1765,6 +1765,7 @@ static void parse_border_style(Object style, FloatConfig *fconfig, Error *err)
     { "double", { "╔", "═", "╗", "║", "╝", "═", "╚", "║" }, false },
     { "single", { "┌", "─", "┐", "│", "┘", "─", "└", "│" }, false },
     { "shadow", { "", "", " ", " ", " ", " ", " ", "" }, true },
+    { "transparent", { " ", " ", " ", " ", " ", " ", " ", " " }, false },
```

Comment:
"Transparent" is not a very good name when the highlight is fully opaque by default. Maybe "solid"?

Reply:
Hmm, but wouldn't people expect to _see_ something when choosing `solid`? When hearing `solid`, I'd rather think `single`. :>

Reply:
They see a solid border.

Reply:
Okay, changed to `solid`. ;]